src/web/app/components/question-types/question-edit-answer-form/num-scale-question-edit-answer-form.component.ts

```diff
@@ -59,4 +59,19 @@ export class NumScaleQuestionEditAnswerFormComponent
     return `${possibleValuesString}]`;
   }
 
+  /**
+   * Checks if the answer value is a valid value in question possible values.
+   */
+  isValidPossibleValue(value: number): boolean {
+    const minValue: number = this.questionDetails.minScale;
+    const maxValue: number = this.questionDetails.maxScale;
+    const increment: number = this.questionDetails.step;
+
+    if (!isNaN(Number(value))) {
+      const currentValue: number = Number(value);
+      return currentValue >= minValue && currentValue <= maxValue && (currentValue - minValue) % increment === 0;
```

Comment:
`(currentValue - minValue) % increment === 0;` will this work for decimal number?

Reply:
I tested it works